resources/integration_test/tests/dbmd_variable_result_sets_function_methods.yml

```diff
@@ -0,0 +1,182 @@
+tests:
+  # getFunctions(catalog, schemaPattern, functionNamePattern)
+  - description: getFunctions_no_filter_returns_all_functions
+    db: integration_test
+    skip_reason: "SQL-669: incorrect column nullability"
+    meta_function: [getFunctions, integration_test, null, '%']
+    expected_result:
+      - [def, BIT_LENGTH, null, 1, returns length of string in bits, BIT_LENGTH]
+      - [def, CAST, null, 1, converts the provided expression into a value of the
+                               specified type., CAST]
+      - [def, CHAR_LENGTH, null, 1, returns length of string, CHAR_LENGTH]
+      - [def, COALESCE, null, 1, 'returns the first non-null value in the list, or
+          null if there are no non-null values.', COALESCE]
+      - [def, CURRENT_TIMESTAMP, null, 1, returns the current date and time., CURRENT_TIMESTAMP]
+      - [def, CURRENT_TIMESTAMP, null, 1, returns the current date and time., CURRENT_TIMESTAMP]
```

Comment:
Overloaded functions are returned multiple times. This seems like valid behavior as this is what we do in the (released) MySQL version of the driver and nothing in the docs says it is incorrect.